nemo/collections/llm/api.py

```diff
@@ -556,6 +556,8 @@ def ptq(
             trainer_kwargs={},
             model_config_overrides={"sequence_parallel": False},
         )
+    if forward_loop is None and not quantization_config.is_weight_only():
```

Comment:
Would you be fine moving it to the original place i.e. `quantize(...)` method? So that possibly "direct" users of `Quantizer` can also benefit from `is_weight_only` check (now it's only available in api.py)